Gems/ImGui/Code/Source/LYCommonMenu/ImGuiLYCommonMenu.cpp

```diff
@@ -571,7 +571,25 @@ namespace ImGui
                 // End LY Common Tools menu
                 ImGui::EndMenu();
             }
+            const int labelSize{ 100 };
+            const int buttonSize{ 40 };
             ImGuiUpdateListenerBus::Broadcast(&IImGuiUpdateListener::OnImGuiMainMenuUpdate);
+            ImGui::SameLine(ImGui::GetWindowContentRegionMax().x - labelSize);
+            float sz = ImGui::GetTextLineHeight() + 3;
+            ImVec2 p = ImGui::GetCursorScreenPos();
+            ImGui::GetWindowDrawList()->AddRectFilled(p, ImVec2(p.x + labelSize, p.y + sz), IM_COL32(0, 115, 187, 255));
+            ImGui::SameLine(ImGui::GetWindowContentRegionMax().x - labelSize);
+            ImGui::SetCursorPosY(ImGui::GetCursorPosY() - 1);
+            ImGui::Text("ImGui:ON");
+            ImGui::SameLine(ImGui::GetWindowContentRegionMax().x - buttonSize);
+            ImGui::PushStyleColor(ImGuiCol_Text, IM_COL32(0, 0, 0, 255));
+            ImGui::PushStyleColor(ImGuiCol_Button, IM_COL32(255, 255, 255, 255));
+            ImGui::PushStyleColor(ImGuiCol_ButtonHovered, IM_COL32(128, 128, 128, 255));
+            if (ImGui::SmallButton("home"))
+            {
+                ImGuiManagerBus::Broadcast(&IImGuiManager::ToggleThroughImGuiVisibleState);
```

Comment:
Hi team, so I think part of the issue here is that UX provided an SVG icon to use in the perspective ImGui header bar when ImGui is active. However in the example above, it shows someone re-created the icon using just boxes and it doesn't show the representation of the Home button being the toggle switch. Now my assumption is that this was done because there was some issues during implementation of the icon and this was not a simple task. 
![image](https://user-images.githubusercontent.com/82551958/124671493-6954d900-de6a-11eb-8dfe-c4c07303d701.png)

Alternatively we would be ok not using the icon and just finding a way to keep the cursor active during ImGui on state and we can then use a toggle button just like other options in the header.

Reply:
The cursor is constrained to the viewport so the button to close has to be there as well.